src/utils/caipConversion.ts

```diff
@@ -25,6 +33,8 @@ export const AvaxCaipId = {
 export const getNetworkCaipId = (network: PartialBy<Network, 'caipId'>) => {
   if (network.caipId) {
     return network.caipId;
+  } else if (network.caip2Id) {
+    return network.caip2Id;
   }
```

Comment:
Why do we have both `caipId` and `caip2Id`? Aren't the two the same?

Reply:
They are...when I was naming the field in Contentful, I made the mistake of naming it `caip2Id` and it propagated through the Proxy API and `@avalabs/core-chains-sdk`.

So far the `caipId` only lived in the extension and was populated programatically by the `decorateWithCaipId(...)` util function:

https://github.com/ava-labs/core-extension/blob/5209e15ce2860ddcf2470617db84ea2032ebf019/src/background/services/network/models.ts#L37

https://github.com/ava-labs/core-extension/blob/876fd7dc0974f6fb16719844ff38659f862afc9a/src/utils/caipConversion.ts#L121-L126

I didn't want to blow these PRs with unnecessary refactorings, so my plan was to just go with it in until the rollout and later get rid of the old field (rename it to `caip2Id` everywhere, since it's also a more fitting name).